UpskillWikipediaUITests/PageObjects/WebViewPage.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+class WebViewPage: BaseAppPage {
+    
+    // MARK: - Elements
+    private lazy var privacyPolicyPageTitle = app.staticTexts["Wikimedia Foundation Privacy Policy"].firstMatch
+    private lazy var termsOfUsePageTitle = app.staticTexts["Wikimedia Foundation Privacy Policy"].firstMatch
```

Comment:
Boths queries looks the same (text is the same) - is that on purpose?

Reply:
My mistake, corrected already